Apps/W1/ShopifyConnector/app/PermissionSets/ShpfyD365BASIC.PermissionSetExtension.al

```diff
@@ -0,0 +1,8 @@
+/// <summary>
+/// Unknown Shpfy D365 BASIC (ID 30100) extends Record D365 BASIC.
+/// </summary>
+permissionsetextension 30100 "Shpfy D365 BASIC" extends "D365 BASIC"
+{
+    IncludedPermissionSets =
+        "Shpfy Permisions";
```

Comment:
This is correct, because the permissions are already in the role of "D365 BASIC"
Previous we use the permission sets in an xml file, because the permission objects didn't exists.
With the XML file, we must also included these permissions, but now with the permission set objects is this not necessary anymore.

Reply:
Updated with commit: "Inderect Permissions"